### PR TITLE
[P]retty [V]alue [C]ommunications

### PR DESCRIPTION
**Pull request by zsteig**
New format for CAN messages to allow integration team to monitor PVC state

### CODE REVIEW

**Comment by mjmagee991** [comment]
[PVC PDO Validation](https://trello.com/c/XjcBbsXh/45-pvc-pdo-validation)

**Review by S-TH3TA-13 — APPROVED** [review]

File: include/PreCharge/PreCharge.hpp [hunk 1]
```diff
@@ -103,6 +101,12 @@ class PreCharge {
      */
     void getMCKey();
 
+    /**
+     * Get the state of all IO instances and update IOStatus
+     *
```
Comment (mjmagee991) [inline include/PreCharge/PreCharge.hpp]: NIT: Delete this extra line

File: src/PreCharge/PreCharge.cpp [hunk 2]
```diff
@@ -257,10 +276,13 @@ uint16_t PreCharge::getObjectDictionarySize() {
 }
 
 void PreCharge::sendChangePDO() {
```
Comment (mjmagee991) [inline src/PreCharge/PreCharge.cpp:278]: This method needs documentation in the `.hpp` file. Also, it should probably be made private.

File: src/PreCharge/PreCharge.cpp [hunk 3]
```diff
@@ -257,10 +276,13 @@ uint16_t PreCharge::getObjectDictionarySize() {
...
-    uint8_t value = (IOStatus << 8) | Statusword;
-    uint8_t payload[1] = {value};
-    IO::CANMessage changePDOMessage(0, 6, payload, false);
+    uint8_t payload[] = {Statusword, 0x00, static_cast<unsigned short>(keyInStatus) << 4 | static_cast<unsigned short>(stoStatus),
+                                           static_cast<unsigned short>(batteryOneOkStatus) << 4  | static_cast<unsigned short>(batteryTwoOkStatus),
+                                           static_cast<unsigned short>(eStopActiveStatus) << 4 | static_cast<unsigned short>(forwardStatus),
+                                           static_cast<unsigned short>(pcStatus) << 4 | static_cast<unsigned short>(dcStatus),
+                                           static_cast<unsigned short>(contStatus) << 4 | static_cast<unsigned short>(apmStatus)};
```
Comment (mjmagee991) [inline src/PreCharge/PreCharge.cpp]:
```suggestion
                                           static_cast<unsigned short>(contStatus) << 4 | static_cast<unsigned short>(apmStatus),};
```
Adding a comma here will fix the auto-formatting

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Looks great. I have a couple minor nitpicks, but they shouldn't take long

**Review by mjmagee991 — APPROVED** [review]
Looks good

**Pull request merged**